packages/desktop-client/src/components/modals/NordigenExternalMsg.js

```diff
@@ -246,20 +259,17 @@ export default function NordigenExternalMsg({
             >
               Success! Click to continue &rarr;
             </Button>
-          ) : isConfigured ? (
+          ) : isConfigured || isNordigenSetupComplete ? (
             renderLinkButton()
           ) : (
-            <P style={{ color: colors.r5 }}>
-              Nordigen integration has not been configured so linking accounts
-              is not available.{' '}
-              <a
-                href="https://actualbudget.github.io/docs/Accounts/connecting-your-bank/"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                Learn more.
-              </a>
-            </P>
+            <>
+              <P style={{ color: colors.r5 }}>
+                Nordigen integration has not yet been configured.
+              </P>
+              <Button primary onClick={onNordigenInit}>
+                Click here to configure
```

Comment:
One small wording suggestion: avoid “click” here in favor of making the button a verb phrase that describes what happens when you click it.

```suggestion
                Configure Nordigen integration
```